typings/resin-sdk.d.ts

```diff
@@ -308,11 +308,13 @@ declare namespace ResinSdk {
 	interface InvoiceInfo {
 		closed_at: string;
 		created_at: string;
+		due_on: string;
 		currency: string;
 		invoice_number: string;
 		subtotal_in_cents: string;
 		total_in_cents: string;
 		uuid: string;
+		state: string;
```

Comment:
Seems that using `'pending' | 'paid' | 'failed' | 'past_due'` could be useful in the UI.